cypress/support/commands.js

```diff
@@ -25,17 +25,44 @@
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
 import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
+import ArticlePageObject from '../support/pages/articlePageObject';
+
+const articlePage = new ArticlePageObject();
 
 addMatchImageSnapshotCommand();
 
 Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+Cypress.Commands.add('register', (
+  email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
   cy.request('POST', '/users', {
     email,
     username,
     password
   });
 });
+
+Cypress.Commands.add('registerAndLogin', (email, username, password) => {
+  cy.register(email, username, password)
+    .then(() => {
+      cy.request('POST', '/users/login', {
+        user: {
+          email,
+          password
+        }
+      }).then((response) => {
+        cy.setCookie('drash_sess', response.body.user.token);
+      });
+    });
+});
+
+Cypress.Commands.add('createArticle', (title, description, body, tag) => {
+  articlePage.visit();
+  articlePage.typeTitle(title);
+  articlePage.typeDescription(description);
+  articlePage.typeBodyArticle(body);
+  articlePage.typeTag(tag);
+  articlePage.clickPublishBtn();
+});
```

Comment:
The `createArticle` command directly interacts with the `articlePage` object. Consider whether this command should be more generic to allow for greater reusability in different contexts or if it's intended to be tightly coupled with the `articlePage`.